.circleci/config.yml

```diff
@@ -90,6 +89,20 @@ jobs:
           paths:
             - /home/circleci/go/pkg/mod
 
+  smoke-test:
+    machine:
+      image: ubuntu-2204:2024.01.1
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v2-googleko-{{ checksum "Makefile" }}
```

Comment:
<img width="2072" alt="Screenshot 2024-09-11 at 2 19 04 PM" src="https://github.com/user-attachments/assets/44603462-5689-4989-8e5d-c437188679ca">
I was hoping circleci would handle that as well. However, based on all previous builds before this PR, it seems that the `run` step always executes, regardless of the cache's state.

Reply:
For future improvements, we can add a command to skip a job if a cache key already exist. That would allow us to only install `ko` when the `Makefile` has changed : https://discuss.circleci.com/t/skip-job-if-cache-exists/28726/8